scripts/builds/react-router-dom.js

```diff
@@ -75,11 +75,22 @@ const webModules = [
     },
     external: ['history', 'prop-types', 'react', 'react-router'],
```

Comment:
We just need to remove prop-types from this line so Rollup doesn't treat it as an external. Then the `ignore` plugin (below) can do its job.

Reply:
No need for `babel-plugin-transform-react-remove-prop-types`